Tasks/DeployVisualStudioTestAgent/DownloadTestAgent.ps1

```diff
@@ -15,35 +15,44 @@ function ValidateSourceFile([string] $sourcePath)
    }
 }
 
-# Create the parent directory if it does not exist
-$destinationDirectory = Split-Path -Path $destinationPath -Parent
-$isPresent = Test-Path $destinationDirectory
-if(!$isPresent)
-{
-    New-Item -ItemType Directory -Path $destinationDirectory
-}
-
-# Check if the given path is a valid Uri
-$isUri = [System.Uri]::IsWellFormedUriString($sourcePath, [System.UriKind]::Absolute)
+$source = $sourcePath.Split(";")
+$counter = 0;
+$destinationFile = $destinationPath.Split(";")
```

Comment:
can we test negative scenarios and see if the error is clear ?
for ex. split returning empty string, invalid syntax etc ?